react/components/SearchBar.js

```diff
@@ -7,42 +7,45 @@ import { Button } from 'vtex.styleguide'
 
 import header from '../store-header.css'
 
-const SearchBar = ({ compactMode, autoFocus, onCancel }) => {
-  return (
-    <React.Fragment>
-      <div className={`${header.topMenuSearchBar} flex pa2-m flex-grow-1 justify-center`}>
-        <div className="w-100 mw7">
-          <Adopt
-            mapper={{
-              placeholder: <FormattedMessage id='header.search-placeholder' />,
-              emptyPlaceholder: <FormattedMessage id='header.search-emptyPlaceholder' />,
-            }}
-          >
-            {({ placeholder, emptyPlaceholder }) => (
-              <ExtensionPoint
-                id="search-bar"
-                placeholder={placeholder}
-                emptyPlaceholder={emptyPlaceholder}
-                compactMode={compactMode}
-                autoFocus={autoFocus}
-              />
-            )}
-          </Adopt>
-        </div>
+const SearchBar = ({ isMobile, autoFocus, onCancel, height, iconClasses }) => (
+  <React.Fragment>
```

Comment:
There's only one div inside this fragment, thus I believe the fragment is not necessary.